src/www/ui/core-schema.dat

```diff
@@ -1241,6 +1241,31 @@
   $Schema["TABLE"]["license_std_comment"]["is_enabled"]["ALTER"] = "ALTER TABLE \"license_std_comment\" ALTER COLUMN \"is_enabled\" SET NOT NULL, ALTER COLUMN \"is_enabled\" SET DEFAULT true";
 
 
+  $Schema["TABLE"]["license_acknowledgement"]["la_pk"]["DESC"] = "COMMENT ON COLUMN \"license_acknowledgement\".\"la_pk\" IS 'Acknowledgement id'";
```

Comment:
Would like to call table as `license_std_acknowledgement` to avoid confusion with current acknowledgement data?